src/main/java/net/dv8tion/jda/api/interactions/commands/build/OptionData.java

```diff
@@ -581,6 +771,13 @@ public static OptionData fromData(@Nonnull DataObject json)
         OptionType type = OptionType.fromKey(json.getInt("type"));
         OptionData option = new OptionData(type, name, description);
         option.setRequired(json.getBoolean("required"));
+        if (type == OptionType.INTEGER || type == OptionType.NUMBER)
+        {
+            if (!json.isNull("min_value"))
+                option.setMinValue(type == OptionType.INTEGER ? json.getLong("min_value") : json.getDouble("min_value"));
+            if (!json.isNull("max_value"))
+                option.setMaxValue(type == OptionType.INTEGER ? json.getLong("max_value") : json.getDouble("max_value"));
```

Comment:
I'm pretty sure this will not call the double/long overload as you would expect, you will have to split it up to get the correct method.